### PR TITLE
docker_container: add "device_cgroup_rules" option

### PR DESCRIPTION
**Pull request by franjurinec**
##### SUMMARY

Adds a new option `device_cgroup_rules` (`HostConfig.DeviceCgroupRules` in the [API](https://docs.docker.com/engine/api/v1.28/#tag/Container/operation/ContainerCreate) `>=v1.28`) - allows specifying a list of cgroup rules (https://docs.docker.com/reference/cli/docker/container/run/#device-cgroup-rule).

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

docker_container

### CODE REVIEW

File: changelogs/fragments/910-docker_container-device_cgroup_rules.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "docker_container - add support for ``device_cgroup_rules``."
```
Comment (felixfontein) [inline changelogs/fragments/910-docker_container-device_cgroup_rules.yml:2]:
```suggestion
  - "docker_container - add support for ``device_cgroup_rules`` (https://github.com/ansible-collections/community.docker/pull/910)."
```

File: tests/integration/targets/docker_container/tasks/tests/options.yml [hunk 2]
```diff
@@ -1143,6 +1143,66 @@
...
+  docker_container:
+    image: "{{ docker_test_image_alpine }}"
+    command: '/bin/sh -c "sleep 10m"'
+    name: "{{ cname }}"
+    state: started
+    force_kill: true
+    device_cgroup_rules:
+    - "c 42:* rmw"
+  register: device_cgroup_rules_1
+  ignore_errors: yes
```
Comment (felixfontein) [inline tests/integration/targets/docker_container/tasks/tests/options.yml:1160]:
```suggestion
  ignore_errors: true
```
Same below, and `no` → `false`.

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Comment by franjurinec** [comment]
Thank you for the feedback @felixfontein! 
Applied all suggested changes - let me know if there is anything else to improve.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@franjurinec thanks a lot for your contribution!

**Pull request merged**